Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageExBase.Placeholder.cs

```diff
@@ -52,6 +50,29 @@ public ImageSource PlaceholderSource
             set { SetValue(PlaceholderSourceProperty, value); }
         }
 
+        private static void PlaceholderSourceChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var control = d as ImageExBase;
+
+            if (e.OldValue == null || e.NewValue == null || !e.OldValue.Equals(e.NewValue))
+            {
+                if (control is RoundImageEx round)
```

Comment:
:( I looked it up and I couldn't find anything about it? It is located in quite a bit of my code, in this and in #1116, so it would be a pain to replace.